plugins/TagFix_IntervalConditional.py

```diff
@@ -0,0 +1,74 @@
+#-*- coding: utf-8 -*-
+
+###########################################################################
+##                                                                       ##
+## Copyright Adrien Pavie 2019                                           ##
+##                                                                       ##
+## This program is free software: you can redistribute it and/or modify  ##
+## it under the terms of the GNU General Public License as published by  ##
+## the Free Software Foundation, either version 3 of the License, or     ##
+## (at your option) any later version.                                   ##
+##                                                                       ##
+## This program is distributed in the hope that it will be useful,       ##
+## but WITHOUT ANY WARRANTY; without even the implied warranty of        ##
+## MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the         ##
+## GNU General Public License for more details.                          ##
+##                                                                       ##
+## You should have received a copy of the GNU General Public License     ##
+## along with this program.  If not, see <http://www.gnu.org/licenses/>. ##
+##                                                                       ##
+###########################################################################
+from plugins.Plugin import Plugin
+from transporthours.main import Main
+
+class TagFix_IntervalConditional(Plugin):
+
+    def init(self, logger):
+        Plugin.init(self, logger)
+        self.errors[32502] = { "item": 3250, "level": 3, "tag": ["value", "fix:chair", "public_transport"], "desc": T_(u"Invalid Conditional Intervals") }
+        self._th = Main()
+
+    def check_tags(self, tags):
+        if "interval" not in tags and "interval:conditional" not in tags:
+            return
+
+        # Analyse all tags related to transport hours (interval, interval:conditional, opening_hours)
+        parsedData = self._th.tagsToHoursObject(tags)
+
+        # Check validity of interval tag
+        if parsedData["defaultInterval"] == "invalid":
+            return { "class": 32502, "subclass": 1, "text": T_(u"Invalid interval tag format") }
+
+        # Check validity of interval:conditional tag
+        if parsedData["otherIntervals"] == "invalid":
+            return { "class": 32502, "subclass": 2, "text": T_(u"Conditional intervals tag is not valid") }
+
+        # Check presence of interval tag if interval:conditional is defined
+        if type(parsedData["otherIntervals"]) is list and parsedData["defaultInterval"] == "unset":
+            return { "class": 32502, "subclass": 3, "text": T_(u"An interval:conditional tag is defined without a default interval tag") }
```

Comment:
@frodrigo  we already have this check in mapcss (cl 9014023, https://github.com/Jungle-Bus/transport_mapcss/blob/master/transport.validator.mapcss#L335) but it totally makes sense here. Do we add a `-osmoseItemClassLevel: none;` to the mapcss one ?

Reply:
It will be better to have it in mapcss, since mapcss code is reusable on other tools.

Reply:
I removed this check